geometry_msgs/TransformStamped rviz_pose
bool freeview_enabled
bool abort
bool pause
bool save_mesh
